deploy/docker/README.md

```diff
@@ -20,13 +21,106 @@ $ docker run -d \
     --net=host \
     --pid=host \
     --privileged \
-    gcr.io/k8s-minikube/localkube-amd64:LOCALKUBE_VERSION \
+    gcr.io/k8s-minikube/localkube-image:${LOCALKUBE_VERSION:-v1.5.3} \
     /localkube start \
     --apiserver-insecure-address=0.0.0.0 \
     --apiserver-insecure-port=8080 \
     --logtostderr=true \
     --containerized
 ```
+
+##### Docker for Mac/Windows
+```console
+# Fix mounting
+$ docker run --rm --volume=/:/rootfs:rw --pid=host --privileged gcr.io/k8s-minikube/localkube-image:${LOCALKUBE_VERSION:-v1.5.3} nsenter --mount=/proc/1/ns/mnt sh -c "if ! df | grep /var/lib/kubelet > /dev/null; then mkdir -p /var/lib/kubelet && mount --bind /var/lib/kubelet /var/lib/kubelet && mount --make-shared /var/lib/kubelet; fi"
```

Comment:
Can you add a note here saying that it needs to be run every time the docker vm boots?